src/dmd/dsymbol.d

```diff
@@ -2441,9 +2441,17 @@ Dsymbol handleSymbolRedeclarations(ref Scope sc, Dsymbol s, Dsymbol s2, ScopeDsy
     auto vd2 = s2.isVarDeclaration(); // existing declaration
     if (vd && vd2)
     {
-        // if one is `static` and the other isn't
-        if ((vd.storage_class ^ vd2.storage_class) & STC.static_)
+        /* if one is `static` and the other isn't, the result is undefined
+         * behavior, C11 6.2.2.7
+         * But, gcc gives an error if the first one was not static,
+         * and makes it static if the first one was static.
```

Comment:
you mean: 
```suggestion
         * and makes it static if the second one was static.
```
?

Reply:
Comment looks correct to me. Everything depends on what the _first_ declaration is.